app/views/signs/show.html.haml

```diff
@@ -16,10 +16,15 @@
       .glosses-container.glosses
         %h2.main_gloss= @sign.gloss_main
         %h2.main_gloss.maori-gloss= @sign.gloss_maori
-      = link_to "#", { class: "card__button add-to-vocab-btn card__button--peach", "data-sign-id": @sign.id } do
-        %span.icon-container
-          %i.fi-plus
-        %span.text-container Add to Vocab Sheet
+      .button-container
+        = link_to "/images/signs/400-400/#{@sign.drawing}", { "data-sign-id": @sign.id } do
+          %span
+            = image_tag "/assets/ui/download-button.svg/", class: "download-button"
+          %span.button-action  Download Drawing
+        = link_to "#", { class: "add-to-vocab-btn sign__button--peach", "data-sign-id": @sign.id } do
```

Comment:
why did the classes on this get changed/removed? I cant see any styles for the sign__button--peach class that looks to have replaced the original card__button--peach class?